network/acp118/handler.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package acp118
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/network/p2p"
+	"github.com/ava-labs/avalanchego/proto/pb/sdk"
+	"github.com/ava-labs/avalanchego/snow/engine/common"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+)
+
+var _ p2p.Handler = (*Handler)(nil)
+
+// Attestor defines whether to a warp message payload should be attested to
+type Attestor interface {
+	Attest(message *warp.UnsignedMessage, justification []byte) (bool, *common.AppError)
```

Comment:
Is the `bool` really needed? I feel like just having `*common.AppError` would be a cleaner implementation. (`nil` meaning that signing is allowed)

Reply:
In the original pass i was returning `(bool, error)`. I felt like it was weird to return an error for expected behavior, but I guess we're going to send back an `AppError` in either scenario.... so maybe it does make sense to drop the `bool`.